webapp/src/components/help.md

```diff
@@ -0,0 +1,119 @@
+- Destination IP: Static IP address of the ODE UDP endpoint to forward messages to
+- Port: UDP Port on the ODE (defaults to 46800)
+- Start: Start date of forwarding rule
+- End: End date of forwarding rule (defaults of 10 year lifespan)
+- Security: Whether security headers are enabled
+- Active: Whether the rule is currently enabled or disabled
+- Delete Button: Select to delete that specific messaging rule
+
+##### Message Forwarding Management
+
+Create message forwarding rules. Enter the following information:
+
+- Destination IP address: ODE static IP address (network accessible from RSU)
+- Message Type
+- Security Header (true/false)
+
+##### Firmware Management
+
+Check for available firmware upgrades and apply them, if available
+
+##### Reboot
+
+Reboot the selected RSU
+
+### RSU Status and Message Counts
+
+On the right side of the interface, there are two menu toggles that open menus with more detailed information about the RSUs. The first is the RSU Status Menu, which shows a full list of all RSUs and their current status. This menu can be toggled on or off by clicking the red X in the corner or by selecting the "Display RSU Status" toggle again. From this menu, users also have the option to print a complete report or an error-specific report for further analysis or record-keeping.
+
+For each device listed, the following information is displayed:
+
+- Location: Roadway and mile marker, select the "location" icon to center the map on the device
+- Online Status: reen means the RSU is online and actively reporting data, yellow indicates the RSU is currently offline but was recently active, and red indicates that RSU is offline and has not reported in some time
+- SCMS Status: Status of SCMS certificates, including the expiration date
+- RSU IP Address
+
+<img src=../icons/rsu_status_menu.png alt="RSU Status Menu Display" width="400"/>
+
+The second menu is the Message Count Menu, which allows users to filter RSU message counts by selecting a
+specific time range and message type. The top date-time picker sets the start of the range, while the bottom
+one sets the end. Below these pickers, a dropdown allows users to choose the type of message they want to track.
+Any change made to the date, time, or message type will automatically update the map and table view to reflect
+the selected parameters.
+
+<img src=../icons/rsu_count_menu.png alt="RSU Count Menu" width="300"/>
+
+A table below the filter menu displays the number of messages received from each RSU within the specified time
+range, regardless of whether the RSU is currently online. The table can be sorted by RSU name, road, or message
+count by clicking on the appropriate column header. Each click toggles between ascending and descending order.
+
+<img src=../icons/rsu_count.png alt="RSU Message Count Table" width="300"/>
+
+The RSU Configuration section enables users to apply configuration changes to multiple RSUs within a selected
```

Comment:
This help menu would be accessible to every user, right? Would it make sense to specify that this feature would only be available to users of operator role?

Reply:
Added a small section on permissions, and clarifications for restricted features